fl4health/utils/losses.py

```diff
@@ -74,7 +80,20 @@ def compute(self) -> Losses:
         num_losses = len(self.losses_list)
         # Compute average checkpoint and backward losses across list
         checkpoint_loss = torch.sum(torch.FloatTensor([losses.checkpoint for losses in self.losses_list])) / num_losses
-        backward_loss = torch.sum(torch.FloatTensor([losses.backward for losses in self.losses_list])) / num_losses
+        backward_loss: Union[torch.Tensor, Dict[str, torch.Tensor]]
```

Comment:
my py was giving me some trouble unless I had these somewhat redundant asserts.

Reply:
This is pretty awkward. Maybe, we make the backward value a dictionary as well and have a default key when there is only "one" backward loss. Something like "backward" maybe? That way basic client and any others can just rely on that being there. Then we don't have to do as much here and the types are "unified"?

Reply:
If we do this, the averaging procedure would be the same as `additional_losses`. We should encapsulate the averaging functionality in that case into a class function.